src/Features/CSharp/Portable/EncapsulateField/CSharpEncapsulateFieldService.cs

```diff
@@ -37,9 +37,15 @@ internal class CSharpEncapsulateFieldService : AbstractEncapsulateFieldService
             }
 
             var tempAnnotation = new SyntaxAnnotation();
-            var newIdentifier = SyntaxFactory.Identifier(originalFieldName)
-                                             .WithTrailingTrivia(declarator.Identifier.TrailingTrivia)
-                                             .WithLeadingTrivia(declarator.Identifier.LeadingTrivia);
+            var escapedName = originalFieldName.EscapeIdentifier();
```

Comment:
I poked around in the EncapsulateFieldService a bit and everywhere we're using `IFieldSymbol.Name` which always reports "a" instead of "\u0061", so it'll be more than just fixing this location to properly preserve "\u0061".

I'll file a separate bug.